salt/states/junos.py

```diff
@@ -455,21 +489,21 @@ def unlock(name):
 
 
 @resultdecorator
-def load(name, **kwargs):
+def load(name, path, **kwargs):
```

Comment:
Same issue as above with name and path

Reply:
reverted via https://github.com/saltstack/salt/pull/57281/commits/682cb0a08baa14e1fa4684163c5588cbda672d25#diff-442286d51776ddefc791bc7f15e42fc2R470